railties/lib/rails/generators/rails/app/templates/config/environments/development.rb.tt

```diff
@@ -87,4 +87,9 @@ Rails.application.configure do
   <%- end -%>
   # Raise error when a before_action's only/except options reference missing actions
   config.action_controller.raise_on_missing_callback_actions = true
+  <%- unless skip_rubocop? -%>
+
+  # Apply autocorrection by RuboCop to files generated by `bin/rails generate`.
+  config.generators.apply_autocorrect_after_generate!
```

Comment:
I'm curious if this is an anti-pattern, where we normally use values (especially booleans) for config.

Reply:
I kinda dig it, given the direct system action it sets up.